server/db/migrate/20190528034230_update_block_reward.rb

```diff
@@ -1,5 +1,7 @@
 class UpdateBlockReward < ActiveRecord::Migration[5.2]
   def up
+    return if Rails.env.test?
```

Comment:
Any problem with test env ?

Reply:
`CkbUtils.miner_reward(epoch)`  will call rpc, but there is no available node on test env

Reply:
It's better to push this to task pool and not block migration.

Reply:
ok, it make sense, I will move to task.